osconfig_tests/test_config/project_config.go

```diff
@@ -0,0 +1,35 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package testconfig
+
+// Project is details of test Project.
+type Project struct {
+	TestProjectID        string
+	ServiceAccountEmail  string
+	TestZone             string
+	ServiceAccountScopes []string
+}
+
+// GetProject ...
+func GetProject(projectID, testZone string) *Project {
+	return &Project{
+		TestProjectID:       projectID,
+		TestZone:            testZone,
+		ServiceAccountEmail: "default",
+		ServiceAccountScopes: []string{
```

Comment:
We're going to want to make this an arg so that we can ensure things like inventory work without a service account, fine for now though.

Reply:
inventory does not use this service account. although it has this information. so, yeah, in future we might want to not provide information that are not required by the different sub-agent